### PR TITLE
dv member photo

### LINKED ISSUE #183: Add member image for Duong Vu
this is an issue

### PR DESCRIPTION
**Pull request by ctc-devops**
this is an issue
closes #183

### CODE REVIEW

File: website/src/app/views/about/members.js [hunk 1]
```diff
@@ -848,6 +856,7 @@ const members = {
       linkedinURL: 'https://www.linkedin.com/in/zoyasoy/',
       imageURL: zoyasoy,
     },
+
```
Comment (Vakhshoori101) [inline website/src/app/views/about/members.js:859]: can you remove this empty line pls :)

**Review by Vakhshoori101 — COMMENTED** [review]
everything looks good! just address that one comment and I can merge

File: website/src/app/views/about/members.js (hunk 1 above)
Comment (duonghv1) [reply website/src/app/views/about/members.js]: Hi, I just remove the line

**Review by Vakhshoori101 — APPROVED** [review]
LGTM

**Pull request merged**